elasticsearch/README.md

```diff
@@ -100,74 +100,74 @@ support multiple versions with minimal changes.
-| `transportPort`                    | The transport port that Kubernetes will use for the service. If you change this you will also need to set [transport port configuration][] in `extraEnvs`                                                                                                 | `9300`                                          |
-| `updateStrategy`                   | The [updateStrategy][] for the StatefulSet. By default Kubernetes will wait for the cluster to be green after upgrading each pod. Setting this to `OnDelete` will allow you to manually delete each pod during upgrades                                   | `RollingUpdate`                                 |
-| `volumeClaimTemplate`              | Configuration for the [volumeClaimTemplate for StatefulSets][]. You will want to adjust the storage (default `30Gi` ) and the `storageClassName` if you are using a different storage class                                                               | see [values.yaml][]                             |
+| Parameter                          | Description                                                                                                                                                                                                                                                                                                       | Default                                         |
+|------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------|
+| `antiAffinityTopologyKey`          | The [anti-affinity][] topology key. By default this will prevent multiple Elasticsearch nodes from running on the same Kubernetes node                                                                                                                                                                            | `kubernetes.io/hostname`                        |
+| `antiAffinity`                     | Setting this to hard enforces the [anti-affinity rules](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity). If it is set to soft it will be done "best effort". Other string values will be ignored. Any object value will used as-is to allow custom configurations. | `hard`                                                                                                                    |
```

Comment:
Can you use Markdown implicit links for consistency with the rest of our Markdown documentation?

```suggestion
| `antiAffinity`                     | Setting this to hard enforces the [anti-affinity rules][]. If it is set to soft it will be done "best effort". Other string values will be ignored. Any object value will used as-is to allow custom configurations. | `hard`                                                                                                                    |
```

https://github.com/elastic/helm-charts/blob/master/elasticsearch/README.md#L391 would just need to be updated to `[anti-affinity rules]: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity`